### PR TITLE
Payment processor config edit - autopay toggle, bank account details

### PR DESCRIPTION
**Pull request by sahilpanjwani**
## Description of the change

> Payment processor config edit - autopay toggle, bank account details

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]()

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by goodgoodwish — COMMENTED** [review]
I'm still learning.

Wondering how could you write so many words (changes) in this PR?

**Comment by sahilpanjwani** [comment]
> I'm still learning.
> 
> Wondering how could you write so many words (changes) in this PR?

All Anurag's work :-)

**Comment by dangdennis** [comment]
> I'm still learning.
> 
> Wondering how could you write so many words (changes) in this PR?

@goodgoodwish and `consolidated.json` is generated

File: requestBodies/EditAccountPaymentProcessorConfig.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Edit Payment Procesor Config",
```
Comment (dangdennis) [inline requestBodies/EditAccountPaymentProcessorConfig.json:2]:
```suggestion
    "description": "Edit Payment Processor Config",
```

File: openapi.json [hunk 2]
```diff
@@ -620,6 +620,42 @@
...
     },
+    "/accounts/{account_id}/payment_processor_config": {
+      "put": {
+        "security": [
+          {
+            "oAuth2": []
+          }
+        ],
+        "tags": ["account"],
+        "summary": "Edit the payment processor config for an existing account",
```
Comment (dangdennis) [inline openapi.json:631]: ah I remember now.

so this is essentially a PUT version of the payment processor config API in the account create call

**Review by dangdennis — APPROVED** [review]

File: openapi.json (hunk 2 above)
Comment (sahilpanjwani) [reply openapi.json:631]: yup exactly, the API changes we just pushed :-)

**Pull request merged**